.github/workflows/deploy-preprod-q1.yml

```
@@ -6,17 +6,17 @@ on:
 
 concurrency: q1
 env:
-  IMAGE: ghcr.io/navikt/familie-integrasjoner-q1:${{ github.sha }}
+  IMAGE: ghcr.io/navikt/familie-integrasjoner:${{ github.sha }}
```

går det bra at vi pusher til samme image fra 2 forskjellige workflows som kan kjøre samtidig?

---

Her har det blitt copy-paste. Usikker, så tenker vi beholder `-q1`
```suggestion
  IMAGE: ghcr.io/navikt/familie-integrasjoner-q1:${{ github.sha }}
```